.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
```

I tend to set it to
```suggestion
      interval: "monthly"
```
since I assumed updates of Github actions are less relevant than updates of Julia dependencies (handled with CompatHelper).